main/src/ca/uwaterloo/flix/language/phase/Eraser.scala

```diff
@@ -7,7 +7,7 @@ import ca.uwaterloo.flix.language.ast.ErasedAst.JType._
 import ca.uwaterloo.flix.language.ast.{ErasedAst, FinalAst}
 import ca.uwaterloo.flix.util.Validation
 import ErasedAst.{Expression => ErasedExp}
-import FinalAst.{Expression => FinalExp}
+import FinalAst.{ConstraintParam, Expression => FinalExp}
```

Comment:
We don't use these renamings anywhere else in the compiler.

Instead we simply write out the full names, e.g. `FinalAst.Expression.Var`.

So for consistency I would request that we do the same here.